<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hegemon Desktop 0.1</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icon/hegemon-logo-dark</normaloff>:/icon/hegemon-logo-dark</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTreeView" name="treeView">
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
        <property name="dragDropMode">
         <enum>QAbstractItemView::DragOnly</enum>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectItems</enum>
        </property>
        <property name="iconSize">
         <size>
          <width>16</width>
          <height>16</height>
         </size>
        </property>
        <property name="indentation">
         <number>10</number>
        </property>
        <attribute name="headerVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="placeholderText">
         <string>Buscar...</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Modo:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_2">
          <property name="text">
           <string>Jer치rquico</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icon/explorer</normaloff>:/icon/explorer</iconset>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton">
          <property name="text">
           <string>Etiquetas</string>
          </property>
          <property name="icon">
           <iconset resource="resources.qrc">
            <normaloff>:/icon/tag</normaloff>:/icon/tag</iconset>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRolapp">
    <property name="title">
     <string>Hegem칩n</string>
    </property>
    <addaction name="actionNuevoBloc"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuRolapp"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNuevoBloc"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionSalir">
   <property name="icon">
    <iconset resource="../../stuff/fatcow32.qrc">
     <normaloff>:/fat32/fatcow-hosting-icons-3800/FatCow_Icons32x32/door_in.png</normaloff>:/fat32/fatcow-hosting-icons-3800/FatCow_Icons32x32/door_in.png</iconset>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="toolTip">
    <string>Cierra el programa</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="icon">
    <iconset resource="../../stuff/fatcow32.qrc">
     <normaloff>:/fat32/fatcow-hosting-icons-3800/FatCow_Icons32x32/boxtrapper.png</normaloff>:/fat32/fatcow-hosting-icons-3800/FatCow_Icons32x32/boxtrapper.png</iconset>
   </property>
   <property name="text">
    <string>Acerca de...</string>
   </property>
   <property name="toolTip">
    <string>Informaci칩n sobre Hegem칩n</string>
   </property>
  </action>
  <action name="actionNuevoBloc">
   <property name="icon">
    <iconset resource="../../stuff/fatcow32.qrc">
     <normaloff>:/fat32/fatcow-hosting-icons-3800/FatCow_Icons32x32/document_notes.png</normaloff>:/fat32/fatcow-hosting-icons-3800/FatCow_Icons32x32/document_notes.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo bloc</string>
   </property>
   <property name="toolTip">
    <string>Crea un nuevo bloc para notas</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionNuevaCarpeta">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/img/folders-add.png</normaloff>:/icon/img/folders-add.png</iconset>
   </property>
   <property name="text">
    <string>Nueva Carpeta</string>
   </property>
  </action>
  <action name="actionNuevoDocumento">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icon/img/folder-add.png</normaloff>:/icon/img/folder-add.png</iconset>
   </property>
   <property name="text">
    <string>Nuevo Documento</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
  <include location="../../stuff/fatcow32.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionSalir</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
